airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/manifest.yaml

```diff
@@ -85,6 +85,7 @@ definitions:
       type: DatetimeBasedCursor
       cursor_datetime_formats:
         - "%Y-%m-%dT%H:%M:%SZ"
+        - "%Y-%m-%dT%H:%M:%S%z"
```

Comment:
Should this cursor format also be added to `audit_logs_stream` and `cursor_incremental_sync`?

Reply:
it failed on ticket_forms stream, so I prefer do not change not affected streams.

Reply:
added to audit_logs_stream and cursor_incremental_sync